docs/source/develop.rst

```diff
@@ -22,6 +22,26 @@ and install it from source::
    cd distributed
    python setup.py install
 
+Using conda, for example::
```

Comment:
Since the Dask docs go through setting up a `conda` environment, could we remove this section in favor of a sentence like "A list of dependencies for `distributed` is maintained in `requirements.txt`, while additional dependencies needed for development are listed in `dev-requirements.txt`"?

Reply:
The intention is to provide a small one-stop-shop for a copy-paste snippet that works, with no doubts or questions about how to use the files.  I understand the value of DRY documentation, so it could be removed for the sake of doc-maintenance and consistency.

Reply:
This is removed from this PR, could be taken up in #3406